ts-sdk/whirlpool/tests/sysvar.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, it, beforeAll } from "vitest";
+import type { SysvarRent } from "@solana/sysvars";
+import { lamports } from "@solana/web3.js";
+import { calculateMinimumBalance } from "../src/sysvar";
+import assert from "assert";
+import { setupAta, setupMint } from "./utils/token";
+import { fetchMint } from "@solana-program/token-2022";
+import { rpc } from "./utils/mockRpc";
+import { getTokenSizeForMint } from "../src/token";
+
+describe("Sysvar", () => {
+  let rent: SysvarRent;
+
+  beforeAll(async () => {
+    rent = {
+      lamportsPerByteYear: lamports(3480n),
+      exemptionThreshold: 2.0,
+      burnPercent: 0,
+    };
+  });
+
+  it("Should calculate the correct minimum balance for a token account", async () => {
+    const mint = await setupMint();
```

Comment:
You can make this test synchronous since you don't need to store/fetch anything in bankrun